playbooks/de-rm-containers-cleanup.yaml

```diff
@@ -83,3 +83,17 @@
       ignore_errors: yes
       become: true
       shell: docker rm -fv etc_config_anon_files_1
+
+- name: Remove old 'etc' project containers
+  hosts: services:ui:image-janitor:consul:!grouper
```

Comment:
Grouper hosts are excluded here because they also use a file in `/etc/docker-compose.yml` and we don't want to stop grouper any time this is run. Otherwise it's intended to be pretty broad, obviously.